src/main/java/net/glowstone/entity/GlowEntity.java

```diff
@@ -207,6 +208,11 @@
      * Whether gravity applies to the entity.
      */
     private boolean gravity = true;
+    /**
+     * Whether friction applies to the entity.
+     */
+    @Setter
+    private boolean friction = true;
```

Comment:
Why only lombokify the Setter?

Reply:
Because a lombokified Getter would be called "isFriction" where we'd want "hasFriction". And the getter isn't used anyway, so right now there's none.

Reply:
Ah okay :)